### ISSUE TITLE
Error to execute on Windows 7 32 bits or 64 bits

### ISSUE THREAD

**Issue by angeltonio** [iwalton3/jellyfin-mpv-shim#75]
I downloaded the latest version, but when I install it on a windows 7 service pack 1. It brings up an api-ms-win-crt-runtime-l1-1-0.dll error ... I make a search on Internet and it seems it was installing the Visual C ++ 2015. I downloaded and installed it, but when I go to run the app it is then an error run desktop.exe I stop working .. and there I don't know what happens. Can somebody help me...If it´s importan i can up the photo with the error...cheers

**Comment by iwalton3** [comment]
Does the `jellyfin-mpv-shim_1.5.10.exe` work? You should be able to run that and then cast to it from a web browser. Note that I don't build on or test on Windows 7, so I'm not surprised it is broken.

**Comment by angeltonio** [comment]
Yes it´s work but only on Windows 10. I tested on two arquitecture and work fine, but i don´t know why that happens on Windows 7. It´s a bad crashed

**Comment by angeltonio** [comment]
Maybe I need a old version??...

**Comment by iwalton3** [comment]
The debug build may be helpful to run: https://iwalton.com/ushare/jellyfin-mpv-shim_1.5.10_debug.exe You could also try following the build instructions here: https://github.com/iwalton3/jellyfin-mpv-shim#building-on-windows 

Windows 7 is end of life, so I would encourage upgrading if possible. Last I checked, you [could still upgrade Windows 7 to Windows 10 for free](https://www.forbes.com/sites/gordonkelly/2020/02/04/how-to-upgrade-to-windows-10-for-free-in-2020/). Linux is also supported if the computer is going to be a dedicated Jellyfin client.

**Comment by angeltonio** [comment]
Yes i know that Windows 10 is free, but the problem is about the hardware, i have a old computer, and i live on Cuba...:( and it´s and you know it´s complicated buy a good machine. I going to try with the debug executable and comment....cheers

**Comment by angeltonio** [comment]
This is the error when I executed the debug program...
![IMG_20200608_202713](https://user-images.githubusercontent.com/46872830/84103475-1d263380-a9e1-11ea-9d9a-4acb8090dccb.jpg)
